app/src/views/DrefApplicationExport/PgaExport/index.tsx

```diff
@@ -130,19 +130,23 @@ function PgaExport() {
                 <div className={styles.bankDetails}>
                     <BlockTextOutput
                         label={strings.nationalSocietyBankName}
+                        invalidText
                     />
                     <BlockTextOutput
                         label={strings.nationalSocietyBankAccountNumber}
+                        invalidText
```

Comment:
same as above, invalidText is not boolean, its type is React.ReactNode so we need to pass null